Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity_description.md

```diff
@@ -0,0 +1,14 @@
+Microsoft Cloud App Security is a Cloud Access Security Broker that supports various deployment modes including log collection, 
+API connectors, and reverse proxy. It provides rich visibility, control over data travel, 
+and sophisticated analytics to identify and combat cyberthreats across all your Microsoft and third-party cloud services.
+
+## Grant Demisto Authorization in Microsoft Cloud App Security API
+To access the Microsoft Management Activity API, you need to grant authorization.
```

Comment:
You have `Microsoft Management Activity API` here, please re-review description files